docs/api/blueprint/attendees.apib

```diff
@@ -73,7 +73,8 @@ Create a new attendee.
                     "country": "IN",
                     "is-checked-in": "false",
                     "attendee-notes": "example",
-                    "pdf-url": "http://example.com"
+                    "pdf-url": "http://example.com",
+                    "ticket-id": "1"
```

Comment:
revert

Reply:
@kushthedude If you explain the problem it would be very helpful